src/buildApplication/buildApplication.ts

```diff
@@ -73,58 +71,40 @@ async function prepareAndBuildWorker(
 		exit(1);
 	}
 
-	const nextJsConfigs = await getNextJsConfigs();
-
-	if (nextJsConfigs.basePath) {
-		cliLog(`Using basePath ${nextJsConfigs.basePath}`);
-	}
+	let generatedFunctionsMaps: DirectoryProcessingResults | undefined;
 
 	const functionsDir = resolve('.vercel', 'output', 'functions');
 	if (!(await validateDir(functionsDir))) {
 		cliLog('No functions detected.');
```

Comment:
@james-elicx what do you think, should we change this to be distinct from the `No functions detected` below?

Something like:
```suggestion
		cliLog('No functions detected (Vercel didn't generate a functions directory).');
```
?